src/inline-assembly.md

```diff
@@ -477,3 +478,135 @@ To avoid undefined behavior, these rules must be followed when using function-sc
   - The compiler is currently unable to detect this due to the way inline assembly is compiled, but may catch and reject this in the future.
 
 > **Note**: As a general rule, the flags covered by `preserves_flags` are those which are *not* preserved when performing a function call.
+
+### Directives Support
+
+Inline ASM supports a subset of the directives supported by both GNU AS and LLVM's internal assembler, given as follows.
+The result of using other directives is assembler-specific (and may cause an error, or may be accepted as-is).
+
+The following directives are guaranteed to be supported by the assembler:
+
+
+- `.2byte`
+- `.4byte`
+- `.8byte`
+- `.byte`
+- `.short`
+- `.word`
+- `.long`
+- `.quad`
+- `.float`
+- `.double`
+- `.octa`
+- `.sleb128`
+- `.uleb128`
+- `.ascii`
+- `.asciz`
+- `.string`
+- `.skip`
+- `.space`
+- `.balign`
+- `.balignl`
+- `.balignw`
+- `.balign`
+- `.balignl`
+- `.balignw`
+- `.fill`
+- `.section`
+- `.pushsection`
+- `.popsection`
+- `.text`
+- `.bss`
+- `.data`
+- `.def`
+- `.endef`
+- `.scl`
+- `.comm`
+- `.lcomm`
+- `.option`
+- `.equ`
+- `.set`
+- `.align`
+- `.inst`
+- `.p2align`
+
```

Comment:
```suggestion
- `.bundle_align_mode`
- `.bundle_lock`
- `.bundle_unlock`
```

Reply:
Also, directives like `.dc`, `.dcb`, and `.ds`. Those simply emit data; if they're supported by LLVM we should include them.